### PR TITLE
fix: change keploy installation url

### PR DESCRIPTION
**Pull request by charankamarapu**
# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Change URL of keploy install script both in open source and enterprise section

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: versioned_docs/version-2.0.0/concepts/installation.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ On MacOS and Windows, additional tools are required for Keploy due to the lack o
 Let's get started by setting up the Keploy alias with this command:
 
 ```bash
- curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+ curl --silent -O -L https://keploy.io/install.sh && source install.sh
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/concepts/installation.md:41]: image

Comment (Sonichigo) [reply versioned_docs/version-2.0.0/concepts/installation.md:41]: Getting rm: keploy.sh issue after installation also the install.sh file does not get removed!

File: versioned_docs/version-2.0.0/keploy-cloud/installation.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ To get started with Keploy Cloud, you will need to install the Keploy Cloud Agen
 Install and Validate the Keploy Cloud Agent by following the steps below:
 
 ```bash
-curl --silent -O --location https://keploy-enterprise.s3.us-west-2.amazonaws.com/releases/latest/assets/keploy-enterprise.sh && source keploy-enterprise.sh
+curl --silent -O -L https://keploy.io/ent/install.sh && source install.sh
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/keploy-cloud/installation.md:47]: image

Same issue with the enterprise one-click

**Review by Sonichigo — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.0.0/concepts/installation.md (hunk 1 above)
Comment (charankamarapu) [reply versioned_docs/version-2.0.0/concepts/installation.md:41]: https://github.com/keploy/keploy/pull/1878

File: versioned_docs/version-2.0.0/keploy-cloud/installation.md (hunk 2 above)
Comment (charankamarapu) [reply versioned_docs/version-2.0.0/keploy-cloud/installation.md:47]: https://github.com/keploy/enterprise/pull/205

**Review by Sonichigo — APPROVED** [review]
LGTM 🚀

**Pull request merged**